src/components/Radio/_Radio.scss

```diff
@@ -0,0 +1,67 @@
+.input {
```

Comment:
We never talked directly about the philosophy behind the root class name for this component.  I feel this class name should be unique to this componet and style inheritance should come from the base styles.  Not from other components.  We should have a call about this.

Reply:
Per our conversation on Monday, 4/19, we will move these reorganization changes to a new JIRA story.